gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileSystemTest.java

```diff
@@ -134,4 +137,27 @@ public void testNullness() throws IOException, NoSuchMethodException, SecurityEx
       tester.testAllPublicInstanceMethods(fs);
     }
   }
+
+  @Test
+  public void testListFiles() throws IOException {
+    try (FileSystem fs = CloudStorageFileSystem.forBucket("bucket")) {
+      List<Path> goodPaths = new ArrayList<>();
+      List<Path> paths = new ArrayList<>();
+      goodPaths.add(fs.getPath("dir/angel"));
+      goodPaths.add(fs.getPath("dir/alone"));
+      paths.add(fs.getPath("dir/dir2/another_angel"));
+      paths.add(fs.getPath("atroot"));
+      paths.addAll(goodPaths);
+      goodPaths.add(fs.getPath("dir/dir2/"));
+      for (Path p : paths) {
```

Comment:
I know I am being annoying but our checkstyle configuration complains about variable names of one letter. I believe that in this very simple loops fixing that issue is not so important but I feel like there's a rationale behind it. If don't feel too annoyed please change to `path` (same issue for the following loop).